tests/integration/components/ember-attacher/hide-on-clickout-test.js

```diff
@@ -33,6 +33,8 @@ test('hides when an element outside the target is clicked', async function(asser
 
   await click('#focus-me');
 
+  await waitUntil(() => isVisible(attachment) === false);
```

Comment:
I found that without this `waitUntile`, this test case sometimes fails.
Referring to [the existing code](https://github.com/kybishop/ember-attacher/blob/master/tests/integration/components/ember-attacher/hide-on-escapekey-test.js#L30), I added it.